### ISSUE TITLE
lazyloaded class not always added

### ISSUE THREAD

**Issue by jameslindeman** [aFarkas/lazysizes#86]
I’ve noticed some images aren’t always getting the lazyloaded class added when others are. Same markup, same page. The ones that don’t get the class, do however get the lazyload class removed.

Here’s the code:
```html
<img src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="  data-srcset="small.jpg, big.jpg 900w" class="lazyload">
```

**Comment by aFarkas** [comment]
Can you say, which exact version you are using. I just realized, that this is indeed a bug with 1.1.0-rc1 and 1.1.0-rc2, but this should have been fixed with 1.1.0-rc3

**Comment by jameslindeman** [comment]
Ah, yes it’s v1.1.0-rc2. Thanks, and great work, this is fantastic! I’ll close this.
